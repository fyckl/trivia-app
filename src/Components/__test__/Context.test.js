// import React from "react"
// import {ContextProvider, Context, useAppContext} from "../Context"
// import { fireEvent, cleanup, screen, render, getByText } from "@testing-library/react"
// import "@testing-library/jest-dom/extend-expect"
// import { Enzyme, shallow, configure, mount } from 'enzyme'
// import Adapter from 'enzyme-adapter-react-16';

// configure({ adapter: new Adapter() });



// test("startQuiz functions correctly", () => {
//     const contextValues = {
//         startQuiz: 
//     }
//     jest
//       .spyOn(Context, 'useAppContext')
//       .mockImplementation(() => contextValues);

// })