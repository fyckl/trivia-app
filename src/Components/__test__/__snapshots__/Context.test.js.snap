// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly to match snapshot 1`] = `""`;
